postprocessing/plevel_interpolation/scripts/run_plevel_simple.py

```diff
@@ -0,0 +1,51 @@
+from netCDF4 import Dataset  
+from plevel_fn import plevel_call, daily_average, join_files, two_daily_average, monthly_average
+import sys
+import os
+import time
+import pdb
+import subprocess
+
+start_time=time.time()
+base_dir=os.environ['GFDL_DATA']
```

Comment:
This is a nice change here. In the other example this was a hard coded link to a path name.